docs/getting-started/server/guide.md

```diff
@@ -134,10 +134,10 @@ After that, rerun the docker compose command from Step 5.
 
 ### SQL Server
 
-In order to support ARM based development environments such as the M1 Macs, we use the
-[Azure SQL Edge](https://hub.docker.com/_/microsoft-azure-sql-edge) docker container instead of a
-normal [Microsoft SQL Server](https://hub.docker.com/_/microsoft-mssql-server) container. It behaves
-mostly identical to a regular SQL Server instance and runs on port 1433.
+For ARM based development environments such as the M1 Macs, we use the
+[Microsoft SQL Server](https://hub.docker.com/_/microsoft-mssql-server) docker container with
+`linux/amd64` platform emulation. It behaves identical to a SQL Server instance without emulation
+and runs on port 1433.
```

Comment:
I think we can remove most of this text since we're back on the original docker container.

Reply:
Fixed in [baf1ccd](https://github.com/bitwarden/contributing-docs/pull/435/commits/baf1ccde2abec95f32974b67a77bca3a9a15a49d)